src/mono/wasi/build/WasiApp.Native.targets

```diff
@@ -60,7 +60,7 @@
     <!-- When Building -->
     <PropertyGroup Condition="'$(WasmBuildingForNestedPublish)' != 'true'">
       <!-- build AOT, only if explicitly requested -->
-      <WasmBuildNative Condition="'$(RunAOTCompilation)' == 'true' and '$(RunAOTCompilationAfterBuild)' == 'true'">true</WasmBuildNative>
```

Comment:
Wouldn't it be simpler to set RunAOTCompilation if PublishAot is set ?

Reply:
We could, but the idea is that we can remove RunAOTCompilation once we no longer support releases that used it.